server/src/main/java/org/elasticsearch/action/admin/indices/datastream/DataStreamsStatsAction.java

```diff
@@ -76,7 +77,9 @@ public DataStreamsStatsAction() {
 
     public static class Request extends BroadcastRequest<Request> {
         public Request() {
-            super((String[]) null);
+            // this doesn't really matter since data stream name resolution isn't affected by IndicesOptions and
+            // a data stream's backing indices are retrieved from its metadata
+            super(null, IndicesOptions.fromOptions(false, true, true, true, false, false, true, false));
```

Comment:
Out of curiosity, since we're trying to get stats from closed indices, should we have a disclaimer that the stats (like max timestamp for instance) could be wrong if indices are closed?

Reply:
Probably. Not sure where best to capture that disclaimer. The docs? We'll probably want to state that anyway, since in the future we will probably only pull timestamp data from the latest index (maybe latest two indices) which could also technically lead to inaccurate timestamps.

Reply:
I think the docs is the best place for it